cfme/infrastructure/host.py

```diff
@@ -140,6 +123,7 @@ def __init__(self, name=None, hostname=None, ip_address=None, custom_ident=None,
                  host_platform=None, ipmi_address=None, mac_address=None, credentials=None,
```

Comment:
Fixed. I thought class attribute would be enough. I haven't heard about hot change of appliance in our framework.